pkgs/applications/misc/electrum-ltc/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, python2Packages
+}:
+
+python2Packages.buildPythonApplication rec {
+  name = "electrum-ltc-${version}";
+  version = "2.6.4.2";
+
+  src = fetchurl {
```

Comment:
It's [not on PyPi](https://pypi.python.org/pypi?%3Aaction=search&term=Electrum-ltc&submit=search) though.